utils/clone_resources.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/env bash
+
+git clone git@github.com:apple/swift.git swift
```

Comment:
If the Swift team goes ahead with this approach, please can https://github.com/apple/swift/pull/29 be changed here too.